<?xml version='1.0' encoding='UTF-8'?>
<sdf version="1.5">
    <world name="default">
        <gui>
            <camera name="user_camera">
                <pose>0 -18 8.0 0 0.523 1.5707</pose>
            </camera>
        </gui>
        <light name="sun" type="directional">
            <cast_shadows>1</cast_shadows>
            <pose>0 0 10 0 -0 0</pose>
            <diffuse>0.8 0.8 0.8 1</diffuse>
            <specular>0.2 0.2 0.2 1</specular>
            <attenuation>
                <range>1000</range>
                <constant>0.9</constant>
                <linear>0.01</linear>
                <quadratic>0.001</quadratic>
            </attenuation>
            <direction>0 0.5 -0.9</direction>
        </light>

        <light name="point_light" type="point">
            <pose>0 -0.377195 6.59868 0 -0 0</pose>
            <diffuse>0.784314 0.784314 0.784314 1</diffuse>
            <specular>0.0980392 0.0980392 0.0980392 1</specular>
            <attenuation>
                <range>50</range>
                <constant>0.8</constant>
                <linear>0</linear>
                <quadratic>0.001</quadratic>
            </attenuation>
            <cast_shadows>0</cast_shadows>
            <direction>0 0 -1</direction>
        </light>

        <include>
            <uri>model://ground_plane</uri>
        </include>
<include><name>obstacle_0</name><pose>3.894676 -3.849291 0 0 0 0</pose><uri>model://trash_bin</uri></include><include><name>obstacle_1</name><pose>5.765347 -2.325459 0 0 0 0</pose><uri>model://fire_hydrant</uri></include><include><name>obstacle_2</name><pose>0.954514 -6.001953 0 0 0 0</pose><uri>model://cafe_table</uri></include><include><name>obstacle_3</name><pose>-0.847504 6.120825 0 0 0 0</pose><uri>model://oak_tree</uri></include><include><name>obstacle_4</name><pose>4.818955 -4.107474 0 0 0 0</pose><uri>model://debris_1_2</uri></include><actor name="obstacle_5"><pose>-2.184666 -5.755442 1.066949 0 0 0</pose><skin><filename>sitting.dae</filename><scale>1.046029</scale></skin><animation name="walking"><filename>stand.dae</filename><scale>1</scale><interpolate_x>true</interpolate_x></animation><plugin name="control" filename="/home/changsik/catkin_ws/src/social_nav_sim/obstacle/animated_obstacle_plugin/lib/libanimated_obstacle_plugin_ros.so"><ignore_obstacles><model>ground_plane</model></ignore_obstacles><is_static>true</is_static></plugin></actor><actor name="obstacle_6"><pose>1.030325 5.442881 1.013286 0 0 0</pose><skin><filename>stand_up.dae</filename><scale>0.993418</scale></skin><animation name="walking"><filename>stand.dae</filename><scale>1</scale><interpolate_x>true</interpolate_x></animation><plugin name="control" filename="/home/changsik/catkin_ws/src/social_nav_sim/obstacle/animated_obstacle_plugin/lib/libanimated_obstacle_plugin_ros.so"><ignore_obstacles><model>ground_plane</model></ignore_obstacles><is_static>true</is_static></plugin></actor><actor name="obstacle_7"><pose>-2.484920 5.151474 1.055903 0 0 0</pose><skin><filename>talk_a.dae</filename><scale>1.035199</scale></skin><animation name="walking"><filename>stand.dae</filename><scale>1</scale><interpolate_x>true</interpolate_x></animation><plugin name="control" filename="/home/changsik/catkin_ws/src/social_nav_sim/obstacle/animated_obstacle_plugin/lib/libanimated_obstacle_plugin_ros.so"><ignore_obstacles><model>ground_plane</model></ignore_obstacles><is_static>true</is_static></plugin></actor><actor name="obstacle_8"><pose>-5.567024 0.461128 1.108596 0 0 0</pose><skin><filename>sit_down.dae</filename><scale>1.086859</scale></skin><animation name="walking"><filename>stand.dae</filename><scale>1</scale><interpolate_x>true</interpolate_x></animation><plugin name="control" filename="/home/changsik/catkin_ws/src/social_nav_sim/obstacle/animated_obstacle_plugin/lib/libanimated_obstacle_plugin_ros.so"><ignore_obstacles><model>ground_plane</model></ignore_obstacles><is_static>true</is_static></plugin></actor><actor name="obstacle_9"><pose>-3.643931 3.754998 1.097957 0 0 0</pose><skin><filename>run.dae</filename><scale>1.076428</scale></skin><animation name="walking"><filename>stand.dae</filename><scale>1</scale><interpolate_x>true</interpolate_x></animation><plugin name="control" filename="/home/changsik/catkin_ws/src/social_nav_sim/obstacle/animated_obstacle_plugin/lib/libanimated_obstacle_plugin_ros.so"><ignore_obstacles><model>ground_plane</model></ignore_obstacles><is_static>true</is_static></plugin></actor><actor name="obstacle_10"><pose>3.042200 -4.829166 0.990582 0 0 0</pose><skin><filename>talk_b.dae</filename><scale>0.971159</scale></skin><animation name="walking"><filename>walk.dae</filename><scale>1</scale><interpolate_x>true</interpolate_x></animation><plugin name="control" filename="/home/changsik/catkin_ws/src/social_nav_sim/obstacle/animated_obstacle_plugin/lib/libanimated_obstacle_plugin_ros.so"><ignore_obstacles><model>ground_plane</model></ignore_obstacles><is_static>false</is_static></plugin></actor><actor name="obstacle_11"><pose>2.273488 5.381549 1.116612 0 0 0</pose><skin><filename>sitting.dae</filename><scale>1.094717</scale></skin><animation name="walking"><filename>walk.dae</filename><scale>1</scale><interpolate_x>true</interpolate_x></animation><plugin name="control" filename="/home/changsik/catkin_ws/src/social_nav_sim/obstacle/animated_obstacle_plugin/lib/libanimated_obstacle_plugin_ros.so"><ignore_obstacles><model>ground_plane</model></ignore_obstacles><is_static>false</is_static></plugin></actor><actor name="obstacle_12"><pose>6.333940 1.204050 0.985670 0 0 0</pose><skin><filename>sitting.dae</filename><scale>0.966343</scale></skin><animation name="walking"><filename>walk.dae</filename><scale>1</scale><interpolate_x>true</interpolate_x></animation><plugin name="control" filename="/home/changsik/catkin_ws/src/social_nav_sim/obstacle/animated_obstacle_plugin/lib/libanimated_obstacle_plugin_ros.so"><ignore_obstacles><model>ground_plane</model></ignore_obstacles><is_static>false</is_static></plugin></actor><actor name="obstacle_13"><pose>-0.162838 -5.897007 1.009651 0 0 0</pose><skin><filename>talk_b.dae</filename><scale>0.989854</scale></skin><animation name="walking"><filename>walk.dae</filename><scale>1</scale><interpolate_x>true</interpolate_x></animation><plugin name="control" filename="/home/changsik/catkin_ws/src/social_nav_sim/obstacle/animated_obstacle_plugin/lib/libanimated_obstacle_plugin_ros.so"><ignore_obstacles><model>ground_plane</model></ignore_obstacles><is_static>false</is_static></plugin></actor><actor name="obstacle_14"><pose>3.484851 4.598377 1.039278 0 0 0</pose><skin><filename>talk_a.dae</filename><scale>1.018900</scale></skin><animation name="walking"><filename>walk.dae</filename><scale>1</scale><interpolate_x>true</interpolate_x></animation><plugin name="control" filename="/home/changsik/catkin_ws/src/social_nav_sim/obstacle/animated_obstacle_plugin/lib/libanimated_obstacle_plugin_ros.so"><ignore_obstacles><model>ground_plane</model></ignore_obstacles><is_static>false</is_static></plugin></actor><actor name="obstacle_15"><pose>-4.825979 -3.545464 1.068653 0 0 0</pose><skin><filename>run.dae</filename><scale>1.047699</scale></skin><animation name="walking"><filename>walk.dae</filename><scale>1</scale><interpolate_x>true</interpolate_x></animation><plugin name="control" filename="/home/changsik/catkin_ws/src/social_nav_sim/obstacle/animated_obstacle_plugin/lib/libanimated_obstacle_plugin_ros.so"><ignore_obstacles><model>ground_plane</model></ignore_obstacles><is_static>false</is_static></plugin></actor><actor name="obstacle_16"><pose>0.238837 6.153939 1.098623 0 0 0</pose><skin><filename>run.dae</filename><scale>1.077081</scale></skin><animation name="walking"><filename>walk.dae</filename><scale>1</scale><interpolate_x>true</interpolate_x></animation><plugin name="control" filename="/home/changsik/catkin_ws/src/social_nav_sim/obstacle/animated_obstacle_plugin/lib/libanimated_obstacle_plugin_ros.so"><ignore_obstacles><model>ground_plane</model></ignore_obstacles><is_static>false</is_static></plugin></actor><actor name="obstacle_17"><pose>-5.353863 -1.082094 0.973051 0 0 0</pose><skin><filename>run.dae</filename><scale>0.953971</scale></skin><animation name="walking"><filename>walk.dae</filename><scale>1</scale><interpolate_x>true</interpolate_x></animation><plugin name="control" filename="/home/changsik/catkin_ws/src/social_nav_sim/obstacle/animated_obstacle_plugin/lib/libanimated_obstacle_plugin_ros.so"><ignore_obstacles><model>ground_plane</model></ignore_obstacles><is_static>false</is_static></plugin></actor><actor name="obstacle_18"><pose>-6.093961 -0.980576 0.921955 0 0 0</pose><skin><filename>sitting.dae</filename><scale>0.903878</scale></skin><animation name="walking"><filename>walk.dae</filename><scale>1</scale><interpolate_x>true</interpolate_x></animation><plugin name="control" filename="/home/changsik/catkin_ws/src/social_nav_sim/obstacle/animated_obstacle_plugin/lib/libanimated_obstacle_plugin_ros.so"><ignore_obstacles><model>ground_plane</model></ignore_obstacles><is_static>false</is_static></plugin></actor><actor name="obstacle_19"><pose>5.475847 -0.335105 1.095451 0 0 0</pose><skin><filename>talk_a.dae</filename><scale>1.073972</scale></skin><animation name="walking"><filename>walk.dae</filename><scale>1</scale><interpolate_x>true</interpolate_x></animation><plugin name="control" filename="/home/changsik/catkin_ws/src/social_nav_sim/obstacle/animated_obstacle_plugin/lib/libanimated_obstacle_plugin_ros.so"><ignore_obstacles><model>ground_plane</model></ignore_obstacles><is_static>false</is_static></plugin></actor><include><name>goal</name><pose>-4.560915 4.428079 0 0 0 0</pose><uri>model://goal_marker</uri></include></world>
</sdf>
